ELASTICSEARCH_DESIGN.md

```diff
@@ -1,51 +1,65 @@
-# Design of the ElasticSearch backend
+# Design of the Elasticsearch backend
 
-This document describes data model for storing and retrieving metadata and timeseries data to/from ElasticSearch
+This document describes data model for storing and retrieving metadata and timeseries data to/from
+Elasticsearch.
 
 
 # Indices
 
 ## Metadata
 
+Metric names are split per component: from `p0` to `p63` (default value).
```

Comment:
Maybe add some text about `name` and `config`  + the fact that they are stored in a set of time based indices.
Maybe add the default index mapping here, we should be pretty self-explanatory.

Reply:
Done.